cds/config.py

```diff
@@ -617,12 +617,15 @@ def _(x):
     'cds_video',
     'cds_embed_video',
     'zip',
+    'cds_deposit_video',
 ]
 # Licence key for THEO player
 THEO_LICENCE_KEY = 'CHANGE_ME'
 # Wowza server URL for m3u8 playlist generation
 WOWZA_PLAYLIST_URL = ('https://wowzaqaedge.cern.ch/cdseos/_definist_/smil:'
                       '{filepath}/playlist.m3u8')
+WOWZA_VIDEO_URL = ('https://wowzaqaedge.cern.ch/cdseos/_definist_/mp4:'
```

Comment:
this is used for Jinja formatter, we have to find a better way to do it on the deposit api